staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -322,12 +327,12 @@ func (e *Store) Create(ctx genericapirequest.Context, obj runtime.Object, includ
 		}
 	}
 	if !includeUninitialized {
-		return e.WaitForInitialized(ctx, out)
+		return e.waitForInitialized(ctx, out, qualifiedResource)
```

Comment:
no need to change the signature, you can just call `e.qualifiedResourceFromContext(ctx)` inside `e.waitForInitialized`